### PR TITLE
add doc colection1

### PR DESCRIPTION
**Pull request by ovidiuasiminei-ssc**
Added :
  -  Config Controller 
  - Firewall Rules Creation
  - IAP access into a VM 
  - Cloud Load Balancing

### CODE REVIEW

File: Architecture/Addendum.md [hunk 1]
```diff
@@ -185,4 +193,52 @@ kpt [installation](https://kpt.dev/installation/)
...
+
+Policy Controller enables the enforcement of fully programmable policies for your clusters. These policies act as "guardrails" and prevent any changes to the configuration of the Kubernetes API from violating security, operational, or compliance controls.
+You can set policies to actively block non-compliant API requests, or simply to audit the configuration of your clusters and report violations. Policy Controller is based on the open source Open Policy Agent Gatekeeper - [OPA ](https://open-policy-agent.github.io/gatekeeper/website/docs/operations/) project and comes with a full library of pre-built policies for common security and compliance controls.
+In addition to actively controlling your Kubernetes environment, you can optionally use Policy Controller as a way to analyze configuration for compliance before deployment. This helps provide valuable feedback during the process of configuration changes and ensures any non-compliant changes are caught early before they might be rejected during application.
+
+### Firewall Rules Creation 
+
+#### [VPC firewall rules overview](https://cloud.google.com/vpc/docs/firewalls)
```
Comment (Michael-McAuley-SSC) [inline Architecture/Addendum.md:215]: Rules Overview

File: Architecture/Addendum.md [hunk 2]
```diff
@@ -185,4 +193,52 @@ kpt [installation](https://kpt.dev/installation/)
...
+- You've configured your VPC to allow IAP traffic to your VM instance.
+- You've downloaded and installed IAP Desktop on your local computer.
+
+#### [IAP TCP forwarding overview](https://cloud.google.com/compute/docs/instances/connecting-to-windows#remote-desktop-connection-app)
+
+IAP's TCP forwarding feature lets you control who can access administrative services like SSH and RDP on your backends from the public internet. The TCP forwarding feature prevents these services from being openly exposed to the internet. Instead, requests to your services must pass authentication and authorization checks before they get to their target resource.
+
+### Cloud Load Balancing 
+
+#### [Cloud Load Balancing overview](https://cloud.google.com/load-balancing/docs/load-balancing-overview)
```
Comment (Michael-McAuley-SSC) [inline Architecture/Addendum.md:232]: caps

File: Architecture/Addendum.md [hunk 3]
```diff
@@ -185,4 +193,52 @@ kpt [installation](https://kpt.dev/installation/)
...
+VPC firewall rules let you allow or deny connections to or from virtual machine (VM) instances in your VPC network. Enabled VPC firewall rules are always enforced, protecting your instances regardless of their configuration and operating system, even if they have not started up.
+
+### IAP access into a VM 
+
+IAP Desktop is a Windows application that lets you manage multiple Remote Desktop connections to Windows VM instances. IAP Desktop connects to VM instances by using Identity-Aware Proxy TCP forwarding and does not require VM instances to have a public IP address.
+Before you connect by using IAP Desktop, make sure that the following prerequisites are met:
+- You've configured your VPC to allow IAP traffic to your VM instance.
+- You've downloaded and installed IAP Desktop on your local computer.
+
+#### [IAP TCP forwarding overview](https://cloud.google.com/compute/docs/instances/connecting-to-windows#remote-desktop-connection-app)
```
Comment (Michael-McAuley-SSC) [inline Architecture/Addendum.md:226]: Overview

File: Architecture/Addendum.md [hunk 4]
```diff
@@ -185,4 +193,52 @@ kpt [installation](https://kpt.dev/installation/)
...
+
+#### [Config Controller overview](https://cloud.google.com/anthos-config-management/docs/concepts/config-controller-overview)
+
+Config Controller provides a managed control plane, based on Kubernetes. In addition, Config Controller instances come pre-installed with Policy Controller, Config Sync, and Config Connector. By using these components, you can leverage the tools and workflows of Kubernetes to manage Google Cloud resources and achieve consistency by using a GitOps workflow. 
+To learn more, see the Config Controller overview.
+
+### Policy controller
+
+#### [Policy controller overview](https://cloud.google.com/anthos-config-management/docs/concepts/policy-controller)
+
```
[the remark this answers is not in the document]
Comment (ovidiuasiminei-ssc) [reply Architecture/Addendum.md:208]: done

File: Architecture/Addendum.md (hunk 1 above)
Comment (ovidiuasiminei-ssc) [reply Architecture/Addendum.md:215]: done

File: Architecture/Addendum.md (hunk 2 above)
Comment (ovidiuasiminei-ssc) [reply Architecture/Addendum.md]: done

File: Architecture/Addendum.md (hunk 3 above)
Comment (ovidiuasiminei-ssc) [reply Architecture/Addendum.md]: done

File: Architecture/Addendum.md [hunk 5]
```diff
@@ -185,4 +193,52 @@ kpt [installation](https://kpt.dev/installation/)
...
+You can set policies to actively block non-compliant API requests, or simply to audit the configuration of your clusters and report violations. Policy Controller is based on the open source Open Policy Agent Gatekeeper - [OPA ](https://open-policy-agent.github.io/gatekeeper/website/docs/operations/) project and comes with a full library of pre-built policies for common security and compliance controls.
+In addition to actively controlling your Kubernetes environment, you can optionally use Policy Controller as a way to analyze configuration for compliance before deployment. This helps provide valuable feedback during the process of configuration changes and ensures any non-compliant changes are caught early before they might be rejected during application.
+
+### Firewall Rules Creation 
+
+#### [VPC firewall rules overview](https://cloud.google.com/vpc/docs/firewalls)
+Virtual Private Cloud (VPC) firewall rules apply to a given project and network. If you want to apply firewall rules to multiple VPC networks in an organization.
```
[the remark this answers is not in the document]
Comment (ovidiuasiminei-ssc) [reply Architecture/Addendum.md:216]: Is not .

File: Architecture/Addendum.md [hunk 6]
```diff
@@ -185,4 +193,52 @@ kpt [installation](https://kpt.dev/installation/)
...
+**Important:**  Please note that kpt is available through the gcloud components, however you may wish to use a specific version, or newer version than that is provided by the gcloud SDK.
+
+### Set up Config Controller 
+
+#### [Config Controller overview](https://cloud.google.com/anthos-config-management/docs/concepts/config-controller-overview)
+
+Config Controller provides a managed control plane, based on Kubernetes. In addition, Config Controller instances come pre-installed with Policy Controller, Config Sync, and Config Connector. By using these components, you can leverage the tools and workflows of Kubernetes to manage Google Cloud resources and achieve consistency by using a GitOps workflow. 
+To learn more, see the Config Controller overview.
+
+### Policy controller
```
[the remark this answers is not in the document]
Comment (ovidiuasiminei-ssc) [reply Architecture/Addendum.md]: done

File: Architecture/Addendum.md [hunk 7]
```diff
@@ -185,4 +193,52 @@ kpt [installation](https://kpt.dev/installation/)
 
 kpt [functions catalog](https://catalog.kpt.dev/?id=curated-functions-catalog)
 
-**Important:**  Please note that kpt is available through the gcloud components, however you may wish to use a specific version, or newer version than that is provided by the gcloud SDK.
\ No newline at end of file
+**Important:**  Please note that kpt is available through the gcloud components, however you may wish to use a specific version, or newer version than that is provided by the gcloud SDK.
+
+### Set up Config Controller 
+
+#### [Config Controller overview](https://cloud.google.com/anthos-config-management/docs/concepts/config-controller-overview)
```
[the remark this answers is not in the document]
Comment (ovidiuasiminei-ssc) [reply Architecture/Addendum.md]: done

File: Architecture/Addendum.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (lucstjean-ssc) [reply Architecture/Addendum.md:216]: I agree with Michael. 

If you want to apply firewall rules to multiple VPC networks in an organization (then what do you do?)

File: Architecture/Addendum.md [hunk 8]
```diff
@@ -185,4 +193,52 @@ kpt [installation](https://kpt.dev/installation/)
...
+In addition to actively controlling your Kubernetes environment, you can optionally use Policy Controller as a way to analyze configuration for compliance before deployment. This helps provide valuable feedback during the process of configuration changes and ensures any non-compliant changes are caught early before they might be rejected during application.
+
+### Firewall Rules Creation 
+
+#### [VPC Firewall Rules Overview](https://cloud.google.com/vpc/docs/firewalls)
+Virtual Private Cloud (VPC) firewall rules apply to a given project and network. If you want to apply firewall rules to multiple VPC networks in an organization.
+VPC firewall rules let you allow or deny connections to or from virtual machine (VM) instances in your VPC network. Enabled VPC firewall rules are always enforced, protecting your instances regardless of their configuration and operating system, even if they have not started up.
+
+### IAP access into a VM
```
Comment (davelanglois-ssc) [inline Architecture/Addendum.md:219]: I think the title should be ... into a windows VM

File: Architecture/Addendum.md [hunk 9]
```diff
@@ -185,4 +193,52 @@ kpt [installation](https://kpt.dev/installation/)
...
+VPC firewall rules let you allow or deny connections to or from virtual machine (VM) instances in your VPC network. Enabled VPC firewall rules are always enforced, protecting your instances regardless of their configuration and operating system, even if they have not started up.
+
+### IAP access into a VM 
+
+IAP Desktop is a Windows application that lets you manage multiple Remote Desktop connections to Windows VM instances. IAP Desktop connects to VM instances by using Identity-Aware Proxy TCP forwarding and does not require VM instances to have a public IP address.
+Before you connect by using IAP Desktop, make sure that the following prerequisites are met:
+- You've configured your VPC to allow IAP traffic to your VM instance.
+- You've downloaded and installed IAP Desktop on your local computer.
+
+#### [IAP TCP Forwarding Overview](https://cloud.google.com/compute/docs/instances/connecting-to-windows#remote-desktop-connection-app)
```
Comment (davelanglois-ssc) [inline Architecture/Addendum.md:226]: this link is for windows vm access and not for tcp forwarding. this link is for iap tcp forwarding https://cloud.google.com/iap/docs/tcp-forwarding-overview

File: Architecture/Addendum.md [hunk 10]
```diff
@@ -185,4 +193,52 @@ kpt [installation](https://kpt.dev/installation/)
 
 kpt [functions catalog](https://catalog.kpt.dev/?id=curated-functions-catalog)
 
-**Important:**  Please note that kpt is available through the gcloud components, however you may wish to use a specific version, or newer version than that is provided by the gcloud SDK.
\ No newline at end of file
+**Important:**  Please note that kpt is available through the gcloud components, however you may wish to use a specific version, or newer version than that is provided by the gcloud SDK.
+
+### Set up Config Controller
```
[the remark this answers is not in the document]
Comment (davelanglois-ssc) [reply Architecture/Addendum.md:198]: adjust table of content

File: Architecture/Addendum.md (hunk 8 above)
Comment (alaincormier-ssc) [reply Architecture/Addendum.md:219]: maybe the titles and/or paragraphs in this section are reversed?

File: Architecture/Addendum.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (ovidiuasiminei-ssc) [reply Architecture/Addendum.md:198]: Thanks, Dave.

File: Architecture/Addendum.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (ovidiuasiminei-ssc) [reply Architecture/Addendum.md]: Fixed .

File: Architecture/Addendum.md (hunk 8 above)
Comment (ovidiuasiminei-ssc) [reply Architecture/Addendum.md]: fixed

File: Architecture/Addendum.md [hunk 11]
```diff
@@ -185,4 +193,53 @@ kpt [installation](https://kpt.dev/installation/)
...
+You can set policies to actively block non-compliant API requests, or simply to audit the configuration of your clusters and report violations. Policy Controller is based on the open source Open Policy Agent Gatekeeper - [OPA ](https://open-policy-agent.github.io/gatekeeper/website/docs/operations/) project and comes with a full library of pre-built policies for common security and compliance controls.
+In addition to actively controlling your Kubernetes environment, you can optionally use Policy Controller as a way to analyze configuration for compliance before deployment. This helps provide valuable feedback during the process of configuration changes and ensures any non-compliant changes are caught early before they might be rejected during application.
+
+### Firewall Rules Creation 
+
+#### [VPC Firewall Rules Overview](https://cloud.google.com/vpc/docs/firewalls)
+
+Virtual Private Cloud (VPC) firewall rules apply to a given project and network. If you want to apply firewall rules to multiple VPC networks in an organization, see [Firewall policies](https://cloud.google.com/vpc/docs/firewall-policies-overview)
```
Comment (lucstjean-ssc) [inline Architecture/Addendum.md:217]: Missing a period (.) at the end of this line :)

Comment (ovidiuasiminei-ssc) [reply Architecture/Addendum.md:217]: OK, Thanks :)

File: Architecture/Addendum.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (davelanglois-ssc) [reply Architecture/Addendum.md]: The original sentence from google cloud documentation is "If you want to apply firewall rules to multiple VPC networks in an organization, see [Firewall policies](https://cloud.google.com/vpc/docs/firewall-policies-overview)". we don't need to provide information on firewall policies as it is not something we use in the landing zone. Please remove that sentence.

[the remark this answers is not in the document]
Comment (ovidiuasiminei-ssc) [reply Architecture/Addendum.md]: done

File: Architecture/Addendum.md [hunk 12]
```diff
@@ -185,4 +193,53 @@ kpt [installation](https://kpt.dev/installation/)
...
+In addition to actively controlling your Kubernetes environment, you can optionally use Policy Controller as a way to analyze configuration for compliance before deployment. This helps provide valuable feedback during the process of configuration changes and ensures any non-compliant changes are caught early before they might be rejected during application.
+
+### Firewall Rules Creation 
+
+#### [VPC Firewall Rules Overview](https://cloud.google.com/vpc/docs/firewalls)
+
+Virtual Private Cloud (VPC) firewall rules apply to a given project and network. If you want to apply firewall rules to multiple VPC networks in an organization, see [Firewall policies](https://cloud.google.com/vpc/docs/firewall-policies-overview).
+VPC firewall rules let you allow or deny connections to or from virtual machine (VM) instances in your VPC network. Enabled VPC firewall rules are always enforced, protecting your instances regardless of their configuration and operating system, even if they have not started up.
+
+### IAP access into a windows VM
```
Comment (davelanglois-ssc) [inline Architecture/Addendum.md:220]: please create a parent section called "Remote Access with IAP" and have the "IAP access into a windows VM" section and the "IAP TCP Forwarding Overview" section as subsections to this new parent

Comment (ovidiuasiminei-ssc) [reply Architecture/Addendum.md]: done

File: Architecture/Addendum.md
[the remark this answers is not in the document]
Comment (ovidiuasiminei-ssc) [reply Architecture/Addendum.md:1]: done

**Review by davelanglois-ssc — APPROVED** [review]

**Review by lucstjean-ssc — APPROVED** [review]
/lgtm

File: Architecture/Addendum.md [hunk 13]
```diff
@@ -44,6 +55,47 @@ The ultimate [infographic on GCP products](https://googlecloudcheatsheet.withgoo
...
+
+[KPT Readme.md](https://github.com/GoogleContainerTools/kpt/#readme) is an open source project used to hydrate yaml, get packages, apply functions, search and replace in yaml manifests. kpt is a package centric toolchain that enables a WYSIWYG configuration authoring and authoring enabling tool.
+
+kpt [installation](https://kpt.dev/installation/)
+
+kpt [functions catalog](https://catalog.kpt.dev/?id=curated-functions-catalog)
+
+**Important:**  Please note that kpt is available through the gcloud components, however you may wish to use a specific version, or newer version than that is provided by the gcloud SDK.
+
+
```
Comment (Michael-McAuley-SSC) [inline Architecture/Addendum.md:98]: extra blank line

File: Architecture/Addendum.md [hunk 14]
```diff
@@ -44,6 +55,47 @@ The ultimate [infographic on GCP products](https://googlecloudcheatsheet.withgoo
 Check your regions latency using [GPCping](https://gcping.com/)
```
Comment (Michael-McAuley-SSC) [inline Architecture/Addendum.md:55]: Missing:
The ultimate [infographic on GCP products](https://googlecloudcheatsheet.withgoo/

**Review by Michael-McAuley-SSC — CHANGES_REQUESTED** [review]
Just that one link that seems to be missing, then can be approved.

File: Architecture/Addendum.md (hunk 14 above)
Comment (ovidiuasiminei-ssc) [reply Architecture/Addendum.md:55]: The link exist.

**Pull request merged**